lib/ansible/config/base.yml

```diff
@@ -1578,7 +1578,7 @@ INVENTORY_CACHE_PLUGIN_PREFIX:
   name: Inventory cache plugin table prefix
   description: The table prefix for the cache plugin. If INVENTORY_CACHE_PLUGIN_PREFIX is not provided CACHE_PLUGIN_PREFIX can be used instead.
   env: [{name: ANSIBLE_INVENTORY_CACHE_PLUGIN_PREFIX}]
-  default: ansible_facts
+  default: ansible_inventory_
```

Comment:
This has always been the default used, the general config docs were wrong and the inventory docs were correct.

Reply:
should we remove the 'settable' part from the config? 
we should be able to as long as all plugins use their own docs, otherwise we should really deprecate the use of the constant

Reply:
also something I should write down: proper process for moving from base.yml to plugin configs

ALLOW_WORLD_READABLE_TMPFILES can be used as example (bad one at times)

Reply:
Removing the settable part is a good idea. Is it worth keeping the entry at all?

Reply:
It appears on the docsite, what i did with the example above was add note to description 'this now lives in the plugins', so those looking for the setting there have a reference.